Dockerfile

```diff
@@ -4,7 +4,9 @@ FROM golang:1.11 as build
 # of downloading from GitHub. All other package dependencies will be downloaded
 # from HEAD.
 ADD . /go/src/github.com/m-lab/epoxy
-RUN CGO_ENABLED=0 go get -v github.com/m-lab/epoxy/cmd/epoxy_boot_server
+RUN go get -t -v github.com/m-lab/epoxy/...
+RUN go test -v github.com/m-lab/epoxy/...
+RUN CGO_ENABLED=0 go get -t -v github.com/m-lab/epoxy/cmd/epoxy_boot_server
```

Comment:
Please put `ENV CGO_ENABLED 0` after the `ADD` line.

That way all the tests will test the same way the code is built.